pkg/translate/translate.go

```diff
@@ -0,0 +1,56 @@
+package translate
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/expect-digital/translate/pkg/convert"
+	"github.com/expect-digital/translate/pkg/model"
+	pb "github.com/expect-digital/translate/pkg/server/translate/v1"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type TranslateServiceServer struct {
+	pb.UnimplementedTranslateServiceServer
+}
+
+func New() *TranslateServiceServer {
+	return new(TranslateServiceServer)
+}
+
+func (t *TranslateServiceServer) UploadTranslationFile(
+	ctx context.Context,
+	req *pb.UploadTranslationFileRequest,
+) (*pb.UploadTranslationFileResponse, error) {
+	var (
+		messages model.Messages
+		err      error
+	)
+
+	switch req.GetType() {
+	default:
+		return nil, status.Errorf(codes.Internal, "")
+	case pb.Type_TYPE_UNSPECIFIED:
```

Comment:
Not good-looking, but it's required by buf-lint to pass. It requires for enum in.proto files to be prefixed with "TYPE_"